### ISSUE TITLE
Add geo-types to default features?

### ISSUE THREAD

**Issue by eseglem** [georust/geojson#118]
Would it make sense to add geo-types as default to the `Cargo.toml` like this?

```
[features]
default = ["geo-types"]
```

This seems like a feature that would be used quite often. It would also be more consistent with the wkt package (see: https://github.com/georust/wkt/blob/master/Cargo.toml#L19 ) if keeping them in line is the intent.

Separate, somewhat related question: Is there a new place to discuss georust now that irc.mozilla.org is being sunset? Rather than opening an issue like this. Was looking earlier and the room there seems pretty dead. I am new to rust, but not to geo, and would like to contribute more if I can, but git issues are not the best way to understand more.

**Comment by frewsxcv** [comment]
> Separate, somewhat related question: Is there a new place to discuss georust now that irc.mozilla.org is being sunset? Rather than opening an issue like this. Was looking earlier and the room there seems pretty dead. I am new to rust, but not to geo, and would like to contribute more if I can, but git issues are not the best way to understand more.

Good call! I just started a Discord server, if that's something you're interested in: https://discord.gg/Fp2aape

**Comment by frewsxcv** [comment]
Also, welcome! Regarding the default feature point, I have a feeling we'll need to make it a required feature anyway. See the conversation in https://github.com/georust/geojson/pull/117 for more info

**Comment by eseglem** [comment]
Yeah it seems likely to be requires for that. Tighter integration between the packages seem necessary, is that the plan?
